Trait-Based View.xctemplate/___FILEBASENAME___.playground/Sources/PlaygroundController.swift

```diff
@@ -26,6 +26,7 @@ public func playgroundController(
   -> UIViewController
 {
   let parent = UIViewController()
+  _ = viewController.view
   parent.view.frame.size = size
   parent.preferredContentSize = parent.view.frame.size
   parent.addChild(viewController)
```

Comment:
I think swapping these two lines fixes things:

``` swift
parent.addChild(viewController)
parent.view.addSubview(viewController.view)
```

To:

``` swift
parent.view.addSubview(viewController.view)
parent.addChild(viewController)
```

Wanna make this change and I can merge? I could have sworn I fixed this before but must've been lost in a conflict.

Reply:
Sure, it's a better fix!
However, the following code makes the playground crash again in my case. Can we mix autolayout with autoresizingmask in the wrapping parrent viewcontroller?

```swift
final class MyViewController: UIViewController {
  override func viewDidLoad() {
    super.viewDidLoad()
    let myView = UIView()
    myView.backgroundColor = .green
    myView.translatesAutoresizingMaskIntoConstraints = false
    myView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
    myView.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
    myView.widthAnchor.constraint(equalToConstant: 200).isActive = true
    myView.heightAnchor.constraint(equalToConstant: 100).isActive = true
    self.view.addSubview(myView)
  }
}
```

Reply:
sorry, my mistake. I should have added the constraints after adding the view. It's working !

Reply:
Definitely down for that!